src/components/CalendarMonth.jsx

```diff
@@ -25,6 +26,7 @@ const propTypes = forbidExtraProps({
   enableOutsideDays: PropTypes.bool,
   modifiers: PropTypes.object,
   orientation: ScrollableOrientationShape,
+  daySize: PropTypes.number,
```

Comment:
Should this allow any number, or only a positive integer? If the latter, let's use airbnb-prop-types to restrict it.